src/pluginlistview.cpp

```diff
@@ -30,6 +33,12 @@ PluginListView::PluginListView(QWidget* parent)
   installEventFilter(new CopyEventFilter(this));
 }
 
+void PluginListView::activated()
+{
+  // update highlighted mods
+  selectionModel()->selectionChanged({}, {});
```

Comment:
I have not tested that but I don't think that's great because it will clear previous selections from the list.

Reply:
It doesn't clear the previous selections because both `selected` and `deselected` are empty, so there's no actual change. I agree that it's not the cleanest solution, but I couldn't move the lambda expression in the `connect` to its own `updateHighlightedMods` function because then I lose access to `mwui->modList`. Do you have any ideas for a better approach?

Reply:
If it does not clear the selection that's fine, but I'm not really fan of that `activated`, isn't there a way to "deduce" that from a Qt signal or something? Also, why isn't there something similar for the data tab?

Reply:
The data tab already had an `activated` function, I also added `ui.tree->selectionModel()->selectionChanged({}, {});` there. Both are called in `MainWindow::on_tabWidget_currentChanged`. It could probably also be done by emitting a signal there and listening for that in both the plugins and data tabs. Should I try that instead?